deb/brave-keyring/DEBIAN/control

```diff
@@ -4,4 +4,4 @@ Maintainer: Ben Kero <bkero@brave.com>
 Priority: optional
 Version: 1.10
 Description: Brave Browser keyring and repository files
-Depends: gnupg, procps
+Depends: gnupg, grep, procps
```

Comment:
Since `grep` is an essential package on Debian, there's [no need to directly depend on it](https://www.debian.org/doc/debian-policy/ch-binary.html#id4).